### ISSUE TITLE
Use a certified open source license

### ISSUE THREAD

**Issue by metasim** [sbt/sbt-native-packager#840]
From the best I can tell from [https://github.com/sbt/sbt-native-packager/blob/master/LICENSE.md](LICENSE.md), this is an open source project in spirit, but not in compliance with the [Open Source Initiative](https://opensource.org/). To ensure continued viability of the project in the corporate space, I'd suggest doing what needs to be done to change the license to an Apache 2.0 or MIT-style license.

**Comment by muuki88** [comment]
@jsuereth any objections?

**Comment by rdgreen** [comment]
Looks like this is the Simplified BSD License, which is already OSI-approved. I think this issue can be closed.

**Comment by metasim** [comment]
@rdgreen I don't agree: unless it explicitly states that it is released under a BSD license (preferably with a link to the source), it won't pass the corporate world sniff test. "Looks like" won't cut it when IP lawyers are involved.
